tests/Behat/features/validation-failure.feature

```diff
@@ -1,52 +0,0 @@
-@javascript
-Feature: Don't lose content when page or block is invalid
```

Comment:
This whole file was testing validation messages after page save. Since we've removed that functionality entire we this feature file is no longer relevant